packages/caliper-core/lib/worker/tx-observers/internal-tx-observer.js

```diff
@@ -34,6 +37,12 @@ class InternalTxObserver extends TxObserverInterface {
     constructor(messenger, managerUuid, workerIndex) {
         super(messenger, workerIndex);
         this.updateInterval = ConfigUtil.get(ConfigUtil.keys.Observer.Internal.Interval);
+        // Check if the updateInterval is not a valid number, and use a default value if needed
+        if (typeof this.updateInterval !== 'number' || isNaN(this.updateInterval) || this.updateInterval < 0) {
+            logger.warn(`Invalid update interval (${this.updateInterval}), using default value of 1000ms`);
```

Comment:
replace `1000ms` with DEFAULT_UPDATE_INTERVAL, ie
using default value of ${DEFAULT_UPDATE_INTERNAL}ms`);